### PR TITLE
Added localization docs for Android

### PR DESCRIPTION
**Pull request by juliangarritano**

### CODE REVIEW

File: source/android/index.md [hunk 1]
```diff
@@ -16,7 +20,7 @@ includes:
   - users/user_data_aside 
   - users/sensitive_user_data
   - users/users_on_multiple_devices
-  - users/authenticating_users
```
Comment (dannytranlx) [inline source/android/index.md]: ?

**Comment by dannytranlx** [comment]
:+1:

File: source/android/index.md (hunk 1 above)
Comment (juliangarritano) [inline source/android/index.md]: Oups. I started this on iOS branch and copied the whole file.
Will fix.

File: source/includes/_localization_android.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Localization
+
+Every string you see in SupportKit can be [customized](#strings-customization) and localized. SupportKit provides a few languages out of the box, but [adding new languages](#adding-more-languages) is easy to do. When localizing strings, SupportKit looks for values in the strings.xml in you app first then in the SupportKit ui bundle, enabling you to customize any strings and add support for other languages.
```
Comment (mspensieri) [inline source/includes/_localization_android.md]: In _your_ app first

**Comment by mspensieri** [comment]
:+1:

File: source/includes/_localization_android.md (hunk 2 above)
Comment (juliangarritano) [inline source/includes/_localization_android.md]: Danke

**Pull request merged**